### PR TITLE
tests for aggregate config store

### PR DESCRIPTION
**Pull request by zachgersh**
Again - just test changes here. Adding tests that were previously not there.

### CODE REVIEW

File: pilot/pkg/config/clusterregistry/conversion.go [hunk 1]
```diff
@@ -67,11 +67,6 @@ func NewClustersStore() *ClusterStore {
 	}
 }
 
-// GetClientAccessConfigs returns map of collected client configs
-//func (cs *ClusterStore) GetClientAccessConfigs() map[string]clientcmdapi.Config {
-//	return cs.clientConfigs
-//}
-
```
Comment (rshriram) [inline pilot/pkg/config/clusterregistry/conversion.go]: why did you remove this?

Comment (zachgersh) [reply pilot/pkg/config/clusterregistry/conversion.go]: we commented out - removing all sorts of unused stuff 💃
